### PR TITLE
Masterbar: move logout to Me tab

### LINKED ISSUE #2493: Move logout to Me page
As part of masterbar changes, we would like to remove the Logout button from `SettingsActivity` and move it to the new `MeActivity` instead. As part of this, let's make the following change in `ActivityLauncher`:

`public static void viewSettings(Activity activity) {` should be updated as `public static void viewSettings(Context context) {`. Since we won't need to use `startActivityForResult` for `SettingsActivity`, we don't need to know about the activity anymore.

### PR DESCRIPTION
**Pull request by nbradbury**
Fix #2493 - moves sign-out and "Help & Support" to the Me tab, removes them from Settings, and fixes a few related issues.

@oguzkocer

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/me/MeFragment.java [hunk 1]
```diff
@@ -101,13 +108,54 @@ private void refreshAccountDetails() {
...
+    private void signoutWithConfirmation() {
+        AlertDialog.Builder dialogBuilder = new AlertDialog.Builder(getActivity());
+        dialogBuilder.setMessage(getString(R.string.sign_out_confirm));
+        dialogBuilder.setPositiveButton(R.string.signout, new DialogInterface.OnClickListener() {
+            public void onClick(DialogInterface dialog, int whichButton) {
+                signout();
+            }
+        });
+        dialogBuilder.setNegativeButton(R.string.cancel, new DialogInterface.OnClickListener() {
+            public void onClick(DialogInterface dialog, int whichButton) {
```
Comment (tonyr59h) [inline WordPress/src/main/java/org/wordpress/android/ui/me/MeFragment.java]: If the button has no action you can just pass null for the OnClickListener and the button will still show up.

File: WordPress/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -7,7 +7,7 @@
     <string name="xmlrpc_error">Couldn\'t connect. Enter the full path to xmlrpc.php on your site and try again.</string>
     <string name="no_network_title">No network available</string>
     <string name="no_network_message">There is no network available</string>
-    <string name="sign_out_confirm">Sign out?</string>
+    <string name="sign_out_confirm">Sign out of WordPress.com?</string>
```
Comment (tonyr59h) [inline WordPress/src/main/res/values/strings.xml]: Should this be updated to match the wording presented on the Me tab (Disconnect)?

File: WordPress/src/main/java/org/wordpress/android/ui/me/MeFragment.java (hunk 1 above)
Comment (nbradbury) [inline WordPress/src/main/java/org/wordpress/android/ui/me/MeFragment.java]: > If the button has no action you can just pass null for the OnClickListener

Good catch! I simply copied the existing code from Settings - I'll correct this.

File: WordPress/src/main/res/values/strings.xml (hunk 2 above)
Comment (nbradbury) [inline WordPress/src/main/res/values/strings.xml]: > Should this be updated to match the wording presented on the Me tab (Disconnect)?

Yes, the wording should be consistent.

**Comment by oguzkocer** [comment]
@nbradbury I just tested the changes and there is a small issue on logout. When you have both a self-hosted site and a WordPress.com account added in the app, if you logout from WordPress.com one from the Me page, it should keep you in the app. When I did that, it showed me the Login page immediately, but I am not sure if it actually removed the self-hosted account or not.

**Comment by nbradbury** [comment]
@oguzkocer I've created #2524 to deal with logging out when the user has both .com and self-hosted.

**Pull request merged**